include/net/buf.h

```diff
@@ -1226,7 +1226,11 @@ struct net_buf *net_buf_alloc_fixed(struct net_buf_pool *pool,
  *
  * @copydetails net_buf_alloc_fixed
  */
-#define net_buf_alloc(pool, timeout) net_buf_alloc_fixed(pool, timeout)
+static inline
+struct net_buf *net_buf_alloc(struct net_buf_pool *pool, k_timeout_t timeout)
```

Comment:
Minor, but is it possible to split this after `*pool,` rather than putting `static inline` on a separate line?

Reply:
Done